src/log-store/build.rs

```diff
@@ -12,13 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#[cfg(all(unix, not(miri)))]
-mod unix;
-// todo(hl): maybe support windows seek_write/seek_read
-#[cfg(any(not(unix), miri))]
-mod fallback;
+use protobuf_build::Builder;
 
-#[cfg(any(all(not(unix), not(windows)), miri))]
-pub use fallback::{pread_exact, pwrite_all};
-#[cfg(all(unix, not(miri)))]
-pub use unix::{pread_exact, pwrite_all};
+fn main() {
```

Comment:
We have already compiles proto files via [prost](https://docs.rs/prost-build/latest/prost_build/) in other module, is it necessary to introduce a new one?

Reply:
raft-engine requires the log Entry implement `protobuf::core::Message`
https://github.com/tikv/raft-engine/blob/618eea7af842a0301945d2998062bb95d0e91aea/src/log_batch.rs#L35-L39

while the message compiled by prost only implement `prost::message::Message`, so the proto files need to be compiled with protobuf's build script.

Reply:
get